jordan_bundy/lib/greet.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var exports = module.exports = {};
+
+var greet = function greet(name) {
+  
+  // Not pretty, but a quick and dirty way to use process.argv
+  var name = name || process.argv;
```

Comment:
True, although name would be undefined if this was run from the command line (for the bonus points).  The question was sort of vague, so this might not have accomplished what it was asking.  My goal was to give my function access to process.argv instead of just running the console.log and passing it in.